README.md

```diff
@@ -47,24 +47,36 @@ React Native boilerplate with mostly used features.
 
 - `yarn lint` Runs eslint checks;
 - `yarn ts-compile-check` Runs TS compiling checks;
+- `yarn rename "New App Title" com.new.appname` Rename application and change package/bundle ids;
 
 ## Getting Started
 
-Environment requirements:
+0. Environment requirements:
+    - NodeJS
+    - Yarn
+    - [Npx](https://nodejs.dev/learn/the-npx-nodejs-package-runner)
+    - [CocoaPods](https://cocoapods.org/)
 
-- NodeJS
-- Yarn
-- [Npx](https://nodejs.dev/learn/the-npx-nodejs-package-runner)
-- [CocoaPods](https://cocoapods.org/)
+1. Clone the repository and create the .env file:
 
-Clone the repository and install the dependencies:
+    ```bash
+    git clone https://github.com/STonkoshkur/react-native-boilerplate.git
```

Comment:
I guess, users which to use boilerplate no need all history of commits, replace this "git clone" with

```bash
git clone --depth 1 https://github.com/STonkoshkur/react-native-boilerplate.git
```